scripts/explorer.js

```diff
@@ -0,0 +1,31 @@
+const modalFile = document.querySelector(".file-explorer");
+
+document
+    .querySelector(".taskbar__file-explorer")
+    .addEventListener("click", () => {
+        if (modalFile.style.display === "block") {
+            modalFile.style.display = "none";
+        } else {
+            modalFile.style.display = "block";
+        }
+    });
+
+document.querySelector(".explorer-trigger").addEventListener("click", () => {
+    if (modalFile.style.display === "block") {
+        modalFile.style.display = "none";
+    } else {
+        modalFile.style.display = "block";
+    }
+});
+
+document.querySelector("#fileExplorer").addEventListener("click", () => {
+    if (modalFile.style.display === "block") {
+        modalFile.style.display = "none";
```

Comment:
infact it might even make sense to give all of the elements that use this functionality a class like 'click-to-hide'  (not sure on naming)
then you could do something like 
```js
document.querySelectorAll('.click-to-hide').forEach( element => element.addEventListener('click', toggleHidden))
```